docs/05-dev-tools/node-rpc/alchemy.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+title: Alchemy
+sidebar_label: Alchemy
+tags: [rsk, rootstock, resources, tutorials,  Blockchains, JSON-RPC, WebSockets, Alchemy, RPC, API, RPC API]
```

Comment:
Include mainnet, testnet in the tags if it is supported. Include this on all the tags in the different tools pages